### ISSUE TITLE
Inconsistent naming convention in `Method` constants

### ISSUE THREAD

**Issue by rcjsuen** [tliron/glsp#38]
https://github.com/tliron/glsp/blob/d7cfc1c7abca7d5d8addb034e99b4cb7c75e5c1e/protocol_3_16/workspace.go#L7

https://github.com/tliron/glsp/blob/d7cfc1c7abca7d5d8addb034e99b4cb7c75e5c1e/protocol_3_16/workspace.go#L42

The same file uses `Method*` and also `Server*`. There are more `Method*` names than `Server*` names so I assume the intent is for all of them to be called `Method*`?

**Comment by tris203** [comment]
Server* names are directed from the Server to the client

**Comment by rcjsuen** [comment]
> Server names are directed from the Server to the client

Sorry, I'm not sure I quite follow that. Could you rephrase that?

**Comment by tris203** [comment]
Methods prefixed with Server are sent from the Server -> Client
Methods prefixed with Method are sent from the Client -> Server
